src/packages/@ncigdc/modern_components/environment.js

```diff
@@ -20,9 +21,12 @@ function fetchQuery(operation, variables, cacheConfig, uploadables) {
   });
 
   const hash = md5(body);
-
   const [componentName] = operation.name.split('_relayQuery');
 
+  if (simpleCache[hash]) {
+    return Promise.resolve(simpleCache[hash]);
+  }
+
   setTimeout(() => reduxStore.dispatch(setLoader(componentName)));
 
   return fetch(
```

Comment:
hmm, the only think that could be different would be `componentName` right? Because relay forces us to have that in the query anyways.